agent-installation/overview.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Overview
+---
+
+This guide provides an overview of the Middleware Agent and instructions for its installation on Kubernetes, Linux, and Docker, and Windows.
+
+The Middleware Agent (MW Agent) is observability software that runs on your host infrastructure collecting events and metrics to send to Middleware for you to analyze and monitor. The MW Agent collects data at the system and process levels in real-time to provide detailed insights into host performance and behavior. 
```

Comment:
"The Middleware Agent (MW Agent) is light weight software that runs on your host infrastructure collecting and sending events and metrics to Middleware."